notes.txt

```diff
@@ -0,0 +1,22 @@
+UI Elements the we need to wireframe -
+
+---> Homepage view of the App
+
+-----> Header (Navbar) with Title, and button to Login 
+-------> On login the User's name is displayed on navbar and button changes to Logout
+
+-----> Main section with card layout for all available movies
+-------> Movie cards are potentially laid out on strips of film.
```

Comment:
I LOVE this idea! This will look awesome.

Reply:
Awesome! I'm stoked to see how you're envisioning the clapBoard. That's a really rad Idea, it makes me think more about the whole aesthetic of the site... It could go a lot of directions right now, but I'm thinking about maybe a shadowy `film-noir` style vibe? or we could make it a little more cartoon-ey like `let's all go to the lobby` style? or? we can do whatever, and I'm really excited!

Reply:
YES! I am totally leaning towards the `film-noir` style vibe. I just see a lot of black, white, grey and pops of red? But We. Can. Play! I am super excited as well.